README.md

```diff
@@ -88,151 +103,149 @@ personInfoDictionary[klaviyo.KLPersonZipDictKey] = "02215"
+* [How to send a push notification campaign](https://help.klaviyo.com/hc/en-us/articles/360006653972)
+* [How to add a push notification to a flow](https://help.klaviyo.com/hc/en-us/articles/12932504108571)
 
-If you would like to track when a user opens a push notification then there is a little more code that you will need to add to your application.
+### Tracking push notifications
 
-In your application delegate, under `application(_:didReceiveRemoteNotification:fetchCompletionHandler:)` add the following:
+The following code example allows you to track when a user opens a push notification.
+
+1. In your application delegate, under `application(_:didReceiveRemoteNotification:fetchCompletionHandler:)` add the following:
 
 ```swift 
-func application(
-    _ application: UIApplication,
-    didReceiveRemoteNotification userInfo: [AnyHashable : Any],
-    fetchCompletionHandler completionHandler: @escaping (UIBackgroundFetchResult) -> Void
-) {
-    if application.applicationState == UIApplication.State.inactive ||
-        application.applicationState == UIApplication.State.background
-    {
-        Klaviyo.sharedInstance.handlePush(userInfo: userInfo as NSDictionary)
-    }
+if application.applicationState == UIApplication.State.inactive || application.application.State == UIApplicationState.background {
+    Klaviyo.sharedInstance.handlePush(userInfo: userInfo as NSDictionary)
+  }
     completionHandler(.noData)
-}
 ```
 
-In addition please add the following code that extends your app delegate:
+2. Add the following code that extends your app delegate:
 
 ```swift
-extension AppDelegate: UNUserNotificationCenterDelegate {
-    func userNotificationCenter(_ center: UNUserNotificationCenter, didReceive response: UNNotificationResponse, withCompletionHandler completionHandler: @escaping () -> Void) {
-        Klaviyo.sharedInstance.handlePush(userInfo: response.notification.request.content.userInfo as NSDictionary)
-        completionHandler()
+    extension AppDelegate: UNUserNotificationCenterDelegate {
+        func userNotificationCenter(_ center: UNUserNotificationCenter, didReceive response: UNNotificationResponse, withCompletionHandler completionHandler: @escaping () -> Void) {
+            Klaviyo.sharedInstance.handlePush(userInfo: response.notification.request.content.userInfo as NSDictionary)
+            completionHandler()
+        }
     }
-}
+    
 ```
 
-That is all you need to do to track opens. Now once your first push notifications have been sent and been opened, you should start to see `Opened Push` metrics within your Klaviyo dashboard.
+Once your first push notifications are sent and opened, you should start to see *Opened Push* metrics within your Klaviyo dashboard.
 
-## [OPTIONAL] Foreground Push Handling
+### Foreground push handling
 
-The code below will enable push notifications to show up when you app is running:
+The following code example allows push notifications to appearshow when your app is running:
```

Comment:
`appearshow`. I assume you want to say just `show` here

Reply:
😢 Yes

Reply:
Gonna re-word.